resonate/coroutine.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import annotations
 
+from abc import ABC
```

Comment:
Not a required change. But having ABC here makes no difference, as far as I understand it makes sense if we were to have `@abstractmethods` which we don't

How about removing it?

Reply:
It does forbid instantiation of LFX/RFX directly I do believe

Reply:
I don't think it does. Unless there's at least one `@abstractmethod` in it. Protocol does prevent instantiation in the other hand...

Reply:
Just tried it and you're right, I will roll this back!